### ISSUE TITLE
Setting Theme throws Exception

### ISSUE THREAD

**Issue by anondev32** [DavBfr/dart_pdf#152]
**Describe the bug**
Throws unhandled exception:
```
[VERBOSE-2:ui_dart_state.cc(148)] Unhandled Exception: NoSuchMethodError: The method 'toDouble' was called on null.
Receiver: null
Tried calling: toDouble()
#0      Object.noSuchMethod (dart:core-patch/object_patch.dart:51:5)
#1      double.* (dart:core-patch/double.dart:36:23)
#2      RichText.layout.<anonymous closure> (package:pdf/widgets/text.dart:574:55)
#3      TextSpan.visitChildren (package:pdf/widgets/text.dart:384:19)
#4      RichText.layout (package:pdf/widgets/text.dart:496:10)
#5      Align.layout (package:pdf/widgets/basic.dart:303:13)
#6      Page.layout (package:pdf/widgets/page.dart:132:13)
#7      Page.postProcess (package:pdf/widgets/page.dart:114:9)
#8      Document.save (package:pdf/widgets/document.dart:67:12)
#9      PdfCreatorNew._writeToFile (package:dvag_app/services/pdf/pdf_creator_new/pdf_creator_new.dart:97:51)
<asynchronous suspension>
#10     PdfCreatorNew.createPdf (package:dvag_app/services/pdf/pdf_creator_new/pdf_creator_new.dart:56:12)
<asynchronous suspension>
#11     PDFDocumentCreator.createPdf (package:dvag_app/services/pdf/pdf_doc_creator.dart:47:26)
<asynchronous suspension>
#12     _ResultsPageState._buildStandardAppBar.<anonymous closure> (package:dvag_app/pages/provider/screens/results_page.dart:144:50)
<asynchronous suspension>
#13     FeatureOverlayWidget.build.<anonymous closure> (package:dvag_app/components/thrid_party/feature_discovery_helper.dart:179:11)
#14     _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:654:14)
#15     _InkResponseState.build.<anonymous closure> (package:flutter/src/material/ink_well.dart:729:32)
#16     GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:182:24)
#17     TapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:365:11)
#18     TapGestureRecognizer.acceptGesture (package:flutter/src/gestures/tap.dart:312:7)
#19     GestureArenaManager.sweep (package:flutter/src/gestures/arena.dart:156:27)
#20     GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:222:20)
#21     GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:198:22)
#22     GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:156:7)
#23     GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:102:7)
#24     GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:86:7)
#25     _rootRunUnary (dart:async/zone.dart:1136:13)
#26     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#27     _CustomZone.runUnaryGuarded (dart:async/zone.dart:931:7)
#28     _invoke1 (dart:ui/hooks.dart:263:10)
#29     _dispatchPointerDataPacket (dart:ui/hooks.dart:172:5)
```

**To Reproduce**
Code snippet to reproduce the behavior:
```dart
    document.addPage(Page(
        pageFormat: PdfPageFormat.a4,
        orientation: PageOrientation.portrait,
        margin: EdgeInsets.all(8.0),
        theme: Theme(
          defaultTextStyle: TextStyle(font: _fontRegular, fontSize: 10.0),
          paragraphStyle: TextStyle(font: _fontRegular, fontSize: 10.0),
          header0: TextStyle(font: _fontRegular, fontSize: 10.0),
          header1: TextStyle(font: _fontRegular, fontSize: 10.0),
          header2: TextStyle(font: _fontRegular, fontSize: 10.0),
          header3: TextStyle(font: _fontRegular, fontSize: 10.0),
          header4: TextStyle(font: _fontRegular, fontSize: 10.0),
          header5: TextStyle(font: _fontRegular, fontSize: 10.0),
          tableHeader: TextStyle(font: _fontRegular, fontSize: 10.0),
          bulletStyle: TextStyle(font: _fontRegular, fontSize: 10.0),
          tableCell: TextStyle(font: _fontRegular, fontSize: 10.0),
        ),
        build: (Context context) {
          return Center(
              child: Text('Text', style: TextStyle(font: _fontRegular, fontSize: 10.0))
          ); // Center
        }));
```
Remove the Theme Code and it works well.
```
    document.addPage(Page(
        pageFormat: PdfPageFormat.a4,
        orientation: PageOrientation.portrait,
        margin: EdgeInsets.all(8.0),
        build: (Context context) {
          return Center(
              child: Text('Text', style: TextStyle(font: _fontRegular, fontSize: 10.0))
          ); // Center
        }));

```

**Expected behavior**
Have it working.

**Screenshots**
Not relevant

**Flutter Doctor**
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.9.1+hotfix.2, on Mac OS X 10.14.6 18G87, locale
    en-DE)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.1)
[✓] Xcode - develop for iOS and macOS (Xcode 10.3)
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.38.1)
[✓] Connected device (1 available)

• No issues found!
```

```
```

**Desktop (please complete the following information):**
 - [X] iOS
 - [ ] Android
 - [ ] Browser

**Smartphone (please complete the following information):**
 - Device: [iPad Air]
 - OS: [iOS 12.4]

**Additional context**

**Comment by DavBfr** [comment]
Fixed in Pdf version 1.3.22

**Comment by DavBfr** [comment]
Don't hesitate to [Buy Me A Coffee](https://www.buymeacoffee.com/JORBmbw9h).
